### PR TITLE
Error message for maps tab at /tags

### LINKED ISSUE #4273: 'Map' tab stays blank if no map is available
### Please describe the problem (or idea)
The `map` tab page at /tags stays blank if no map is available. 

> What happened just before the problem occurred? Or what problem could this idea solve?

screenshot 104

> What did you expect to see that you didn't?

I think we can add a line which says 
`Sorry, no maps tagged with x and y tags are available. You can view other maps at publiclab.org/maps or you may help us add a map for these tags. Thanks`

### Please show us where to look

https://publiclab.org/maps/tag/workshop

### What's your PublicLab.org username?

oorjit_chowdhary

### Browser, version, and operating system
Chrome, Windows 10

****

## Thank you!

Your help makes Public Lab better! We *deeply* appreciate your helping refine and improve this site.

To learn how to write really great issues, which increases the chances they'll be resolved, see:

https://publiclab.org/wiki/developers#Contributing+for+non-coders

### PR DESCRIPTION
**Pull request by oorjitchowdhary**
Fixes #4273 
@publiclab/reviewers Is this good enough?
Here's the error message

screenshot 115

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by oorjitchowdhary** [comment]
I think I should change the sentence to singular as you can search for only one tag at a time.

**Comment by oorjitchowdhary** [comment]
@SidharthBansal Can we merge this?

File: app/views/tag/show/_tab_content.html.erb [hunk 1]
```diff
@@ -46,7 +46,11 @@
 
   <% if @node_type == "maps" %>
     <div class="tab-pane active" id="wikis">
-      <%= render :partial => "map/maps" %>
+     <% if @maps.nil? %>
+        <p><strong>Sorry, no map tagged with this tag is available. You can view other maps <a href="/maps">here</a>.</strong></p>
```
Comment (gauravano) [inline app/views/tag/show/_tab_content.html.erb]:
```suggestion
        <p><strong>Sorry, no map is tagged with `<%= params[:id] %>` tag. You can view other maps <a href="/maps">here</a>.</strong></p>
```

Comment (gauravano) [reply app/views/tag/show/_tab_content.html.erb]: How about this?

Comment (oorjitchowdhary) [reply app/views/tag/show/_tab_content.html.erb]: Oh, that looks wonderful.. I'll add it

**Comment by gauravano** [comment]
Don't forget to add new screen shot. Thanks!

**Comment by oorjitchowdhary** [comment]
screenshot 117

**Comment by oorjitchowdhary** [comment]
Done!

File: app/views/tag/show/_tab_content.html.erb [hunk 2]
```diff
@@ -46,7 +46,11 @@
 
   <% if @node_type == "maps" %>
     <div class="tab-pane active" id="wikis">
-      <%= render :partial => "map/maps" %>
+     <% if @maps.nil? %>
```
Comment (gauravano) [inline app/views/tag/show/_tab_content.html.erb]:
```suggestion
     <% if @nodes.nil? %>
```

It should be @nodes. If you test the route for valid tag, it's still giving error.

Comment (oorjitchowdhary) [reply app/views/tag/show/_tab_content.html.erb]: done!

**Comment by oorjitchowdhary** [comment]
@gauravano I've made all the requested changes.. Please take a look here... Thanks

**Comment by oorjitchowdhary** [comment]
@publiclab/reviewers

**Comment by jywarren** [comment]
This looks great!

**Comment by gauravano** [comment]
Nice work @oorjitchowdhary :tada: :100:!

**Pull request merged**